package.json

```diff
@@ -91,6 +91,7 @@
     "jest": "^26.6.3",
     "jest-image-snapshot": "^4.4.0",
     "jest-puppeteer-docker": "^1.4.2",
+    "jest-useragent-mock": "^0.1.1",
```

Comment:
Do we need to have a dependency just for this? It is pretty easy to mock user Agent (See https://github.com/aws/amazon-chime-sdk-js/blob/main/test/browserbehavior/DefaultBrowserBehavior.test.ts#L59)

Reply:
Hmm I actually tried that before, but kept getting `TypeError: Cannot set property userAgent of [object Navigator] which has only a getter` even when doing `@ts-ignore`. I tried a few other approaches too but didn't seem to work :/ Open to recommendations/thoughts.

Reply:
Regarding this you probably need to override the browser implementation of navigator (In JS SDK we have DOMMockBuilder object https://github.com/aws/amazon-chime-sdk-js/blob/main/test/browserbehavior/DefaultBrowserBehavior.test.ts#L8) .
Alternatively I think you can do something like suggested here https://stackoverflow.com/questions/1307013/mocking-a-useragent-in-javascript.

Reply:
I actually tried the StackOverflow suggestion and I got a similar error (i.e. `__defineGetter__` does not exist). I can go ahead and override the implementation of navigator if that is the preferred approach by the team. I think this is easier / simpler to understand for developers though.

Reply:
The problem is the dependency is version 0 which means unstable and can have a lot of breaking changes. Also there are not many of downloads from that library so not sure how well supported that is.
Lastly, more dependencies means we have to keep update them -> more maintenance efforts. You probably have not seen the large amount of patches we have to do for all these dependencies :)

Reply:
See changes in https://github.com/aws/amazon-chime-sdk-component-library-react/pull/747